### PR TITLE
Add privacy policy

### PR DESCRIPTION
**Pull request by CarlSargunar**
In prep for setting up a newsletter, we need a privacy policy - here's a very basic one I've put together

### CODE REVIEW

File: src/routes/privacy-policy/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+			Under UK data protection law, we must have a “lawful basis” for collecting and using your
+			personal information. There is a list of possible lawful bases in the UK GDPR. You can find
+			out more about lawful bases on the <a href="https://ico.org.uk">ICO’s website</a>.
+		</p>
+		<p>
+			To make a data protection rights request, please contact us using the contact details at the
+			top of this privacy notice.
+		</p>
+
+		<h2>Our lawful bases</h2>
```
Comment (dynamictulip) [inline src/routes/privacy-policy/+page.svelte:43]: For consistency (and a11y), we should either make this one a `h3` subheading or add an id and link in the table of contents

File: src/routes/privacy-policy/+page.svelte
Comment (dynamictulip) [inline src/routes/privacy-policy/+page.svelte:1]: Some extra thoughts:

- should we emphasise somewhere that we do not sell personal data or pass it on to sponsors?
- should we include a section about how to ask us to remove information? And/or mention about unsubscribing from the mailing list?

File: src/routes/privacy-policy/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+		</ul>
+
+		<h2 id="contact-details">Contact details</h2>
+		<p>Email: <a href="mailto:management@dddsouthwest.com">management@dddsouthwest.com</a></p>
+
+		<h2 id="what-information">What information we collect, use, and why</h2>
+		<p>We collect or use the following personal information to provide updates and share news:</p>
+		<ul>
+			<li>Names</li>
+			<li>Email address</li>
```
Comment (dynamictulip) [inline src/routes/privacy-policy/+page.svelte:29]: When signing up for a ticket last year, we also collected information about gender (for anonymised metrics) and allergies and access needs (for supporting attendees on the day)

**Review by dynamictulip — APPROVED** [review]
This is super awesome!

I've added some extra thoughts but this is miles better than the nothing we have currently!

File: src/routes/privacy-policy/+page.svelte (hunk 1 above)
Comment (CarlSargunar) [reply src/routes/privacy-policy/+page.svelte:43]: Good shout!

File: src/routes/privacy-policy/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+	import { pageTitle } from '../../stores.js';
+	pageTitle.set('Privacy Policy');
+</script>
+
+<div class="secondary-bg">
+	<div class="section">
+		<h1>Privacy Policy</h1>
+		<p class="emphasis">
+			This privacy notice tells you what to expect us to do with your personal information. We only
+			hold this information to support our events and for our newsletter. We do not sell, not do or
```
Comment (dynamictulip) [inline src/routes/privacy-policy/+page.svelte:11]:
```suggestion
			hold this information to support our events and for our newsletter. We do not sell or
```

**Review by dynamictulip — APPROVED** [review]
Small typo 😉

**Review by dynamictulip — APPROVED** [review]

**Pull request merged**